test/utils/DefaultAccessControlLateInit.test.ts

```diff
@@ -0,0 +1,88 @@
+import hre from "hardhat";
+import { ethers, deployments } from "hardhat";
+import { contract } from "../library/setup";
+import { ERC20RootVault } from "../types/ERC20RootVault";
+import { expect } from "chai";
+import { DefaultAccessControlLateInit } from "../types";
+
+type CustomContext = {
+    defaultAccessControlLateInit: DefaultAccessControlLateInit;
+};
+
+type DeployOptions = {};
+
+contract<ERC20RootVault, DeployOptions, CustomContext>(
+    "DefaultAccessControlLateInit",
+    function () {
+        before(async () => {
+            this.deploymentFixture = deployments.createFixture(
+                async (_, __?: DeployOptions) => {
+                    const { deployments, getNamedAccounts } = hre;
+                    const { deploy } = deployments;
+                    const { deployer } = await getNamedAccounts();
+
+                    await deploy("DefaultAccessControlLateInit", {
+                        from: deployer,
+                        args: [],
+                        log: true,
+                        autoMine: true,
+                    });
+
+                    this.defaultAccessControlLateInit =
```

Comment:
use this.subject instead of that

it'll be much comfortable to use it in behaviors